### PR TITLE
Ansible modules for PowerFlex release version 1.9.0

### PR DESCRIPTION
**Pull request by kuttattz**
# Description
- Added Ansible role to support installation and uninstallation of Gateway.
- Added Ansible role to support installation and uninstallation of SDR.
- Added Ansible role to support installation and uninstallation of Web UI.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [X] I have performed Ansible Sanity test using --docker default
- [X] I have verified that new and existing unit tests pass locally with my changes
- [X] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [X] Functional testing
- [X] Regression testing
- [X] Unit testing

### CODE REVIEW

**Review by Jennifer-John — APPROVED** [review]

File: playbooks/roles/uninstall_powerflex.yml [hunk 1]
```diff
@@ -17,13 +17,27 @@
...
+    powerflex_sdr_state: absent
+
 - name: Unistall PowerFlex LIA
   hosts: lia
   roles:
     - powerflex_lia
   vars:
     powerflex_lia_state: absent
 
+- name: Unistall PowerFlex web UI
```
Comment (Bhavneet-Sharma) [inline playbooks/roles/uninstall_powerflex.yml:34]: Uninstall

File: playbooks/roles/uninstall_powerflex.yml [hunk 2]
```diff
@@ -17,13 +17,27 @@
...
+  hosts: webui
+  roles:
+    - powerflex_webui
+  vars:
+    powerflex_webui_state: absent
+
 - name: Install and configure PowerFlex TB
```
Comment (Bhavneet-Sharma) [inline playbooks/roles/uninstall_powerflex.yml:41]: Change name as per Uninstall task

File: playbooks/roles/uninstall_powerflex.yml [hunk 3]
```diff
@@ -33,6 +47,13 @@
...
     powerflex_tb_state: absent
 
+- name: Uninstall PowerFlex gateway
+  hosts: gateway
+  roles:
+    - powerflex_gateway
+  vars:
+    powerflex_gateway_state: absent
+
 - name: Unistall PowerFlex MDM
```
Comment (Bhavneet-Sharma) [inline playbooks/roles/uninstall_powerflex.yml:57]: Uninstall

File: playbooks/roles/uninstall_powerflex.yml [hunk 4]
```diff
@@ -17,13 +17,27 @@
...
     powerflex_sds_state: absent
 
+- name: Uninstall PowerFlex SDR
+  hosts: sdr
+  roles:
+    - powerflex_sdr
+  vars:
+    powerflex_sdr_state: absent
+
 - name: Unistall PowerFlex LIA
```
Comment (Bhavneet-Sharma) [inline playbooks/roles/uninstall_powerflex.yml:27]: Uninstall

File: playbooks/roles/uninstall_powerflex.yml [hunk 5]
```diff
@@ -17,13 +17,27 @@
   vars:
```
Comment (Bhavneet-Sharma) [inline playbooks/roles/uninstall_powerflex.yml:17]: fix typo with Uninstall in above tasks as well

**Review by Bhavneet-Sharma — APPROVED** [review]

**Review by trisha-dell — APPROVED** [review]

**Review by Bhavneet-Sharma — APPROVED** [review]

**Pull request merged**